spec/models/article_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe Article, type: :model do
+  describe 'DB table' do
+    it { is_expected.to have_db_column :id }
+    it { is_expected.to have_db_column :header }
+    it { is_expected.to have_db_column :subheader }
+    it { is_expected.to have_db_column :body }
+  end
+
+  describe 'Validations' do
+    it { is_expected.to validate_presence_of :header }
+    it { is_expected.to validate_presence_of :subheader }
+    it { is_expected.to validate_presence_of :body } 
+  end
+
+  describe 'Factory' do
+    it 'should have valid Factory' do
+      expect(FactoryBot.create(:article)).to be_valid 
```

Comment:
we have the world helper here so you can just use `create(:article)`(no need for `FactoryBot`)